package.json

```diff
@@ -39,9 +40,10 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
-    "electron": "3.1.6",
+    "electron": "4.0.8",
     "electron-builder": "^19.43.3",
     "electron-publisher-s3": "^19.53.4",
+    "electron-forge": "5.2.3",
```

Comment:
Looks like this pulled in a lot of dependencies. Not a problem, but surprising. Do you know if there is any way to avoid some of these?

Reply:
I think they're all dependencies so we need them to run the electron-forge. I'll try removing electron-forge and trying with just the electron-packager, which is mentioned specifically in the second github issue linked above.

Reply:
Okay, that didn't work. It definitely is a lot of dependencies to pull in, but I also think it's a matter of time before we need to pull these in. Electron is currently on v8.x.x. and until recently we were on v2.x.x and I've been trying to inch us forward in versions as needed. I think we might end up with these extras as we make that transition.

I'm happy to try other ways to avoid bringing in these dependencies, but I don't know of any. Let me know if you have any ideas!

Reply:
It was worth a quick try, but definitely not worth blocking this whole change :) Sounds like we've got a lot of goodness being added here.